ls1intum/Artemis
# `Development`: Use artemisApp as consistent prefix for translation keys

### Checklist
#### General
#### Client

### Motivation and Context
@Santia-go noticed that modeling exercises don't use the artemisApp prefix for its translation key. This PR fixes this inconsistency for multiple cases.

### Description
There are still a few files left that don't start with artemisApp. These are files like home, error, alert, etc. which are not direct features of Artemis. All other files that refer to Artemis features should now use the artemisApp prefix.

### Steps for Testing
1) Pease review the codebase to check for places where I could have missed some translations. 

2) Test the following components and check if all strings are correctly displayed in both German and English. 

- User Management
- Organisation Management
- Modeling Exercises (the whole lifecycle, participating, assessment, complaints)
- Quiz-Statistics


### Review Progress





#### Code Review
#### Manual Tests
